### DISCUSSION TITLE
Line Chart Customization

### DISCUSSION THREAD

**Discussion by MTPL0005-AbhishekDube** [Abhinandan-Kushwaha/react-native-gifted-charts#848] [Q&A]
Is it possible to add both a value and a label to the y-axis, where the label can be an icon, and can the data points also have custom icons?

**Comment by ak-spotter** [comment]
@MTPL0005-AbhishekDube Labels on the Y-axis can only be strings, but custom data points are supported using the `customDataPoint` prop. See [data-points-related-props](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/blob/master/docs/LineChart/LineChartProps.md#data-points-related-props)
